scripts/shared/validate.sh

```diff
@@ -10,3 +10,5 @@ PACKAGES="$(find_go_pkg_dirs) *.go"
 golangci-lint linters
 
 golangci-lint run --timeout 5m $@
+
+markdownlint -c .markdownlint.yml -i vendor .
```

Comment:
Perhaps only run it if `.markdownlint.yml` file exists?

```
Cannot read or parse config file .markdownlint.yml: ENOENT: no such file or directory, open '.markdownlint.yml'
```

Reply:
It doesn't fail if the config is missing, it just shows a warning and runs with the default.